frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,54 @@
 import React from 'react'
 //Import components that you create here
+import TeamCard from './TeamCard/TeamCard';
+import { Container, Row, Col } from 'reactstrap';
+import './LandingPage.css';
 
 function LandingPage(){
     return(
         //Use components that you import here
-        <></>
+        <>
+        <h1 className = "heading">Meet Our Team!</h1> <br />
```

Comment:
What do you think about putting all of the cards inside of another component and then importing that component here instead. That way the landing page component looks a little cleaner.

Reply:
I can do that! Will push soon